lib/galaxy/job_execution/output_collect.py

```diff
@@ -593,15 +593,16 @@ def default_exit_code_file(files_dir, id_tag):
 
 def collect_extra_files(object_store, dataset, job_working_directory):
     file_name = dataset.dataset.extra_files_path_name_from(object_store)
-    temp_file_path = os.path.join(job_working_directory, "working", file_name)
+    output_dir = "working" if os.path.exists(os.path.join(job_working_directory, "working", file_name)) else "outputs"
```

Comment:
I was going to augment tools with information about how to annotate this 😆 - a whole day project. This is so much easier, really awesome!

Reply:
We should ultimately switch everything, and then keep that logic in for a release or two :).